theme/colors/light.css

```diff
@@ -144,24 +144,27 @@
 :root:-moz-window-inactive {
 	--gnome-headerbar-background: var(--gnome-window-background);
 	--gnome-tabbar-tab-hover-background: #f3f3f3; /* Hardcoded color */
-	--gnome-tabbar-tab-active-background: #EFEFEF; /* Hardcoded color */
+	--gnome-tabbar-tab-active-background: #f0f0f0; /* Hardcoded color */
 }
 
 /* Private colors */
 :root[privatebrowsingmode="temporary"] {
 	--gnome-accent-fg: var(--gnome-palette-blue-4);
 	/* Headerbar */
-	--gnome-headerbar-background: #D7E3F0 !important;
+	--gnome-headerbar-background: #eaf2fc !important;
 	/* Tabs */
-	--gnome-tabbar-tab-hover-background: #cbd7e3; /* Hardcoded color */
-	--gnome-tabbar-tab-active-background: #c6d1dd; /* Hardcoded color */
+	--gnome-tabbar-tab-hover-background: #dde5ee; /* Hardcoded color */
+	--gnome-tabbar-tab-active-background: #d8dfe8; /* Hardcoded color */
 	--gnome-tabbar-tab-active-background-contrast: #a9b6c4; /* Hardcoded color */
-	--gnome-tabbar-tab-active-hover-background: #c0cbd7; /* Hardcoded color */
+	--gnome-tabbar-tab-active-hover-background: #d1d8e1; /* Hardcoded color */
+	/* Buttons */
+	--gnome-button-background: #d7dfe8; /* Hardcoded color */
```

Comment:
Buttons color should be transparent, default ones should be good for private window. This same color is used for entries backgrounds. Any reason for this change?